### PR TITLE
chore(tab): add tests against beta

### PR DESCRIPTION
**Pull request by moriahmaney**

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit 27825a8eca775e25fb806e8a1362483976b95945

https://deploy-preview-755--hopeful-hamilton-4090b9.netlify.com

File: Jenkinsfile.tab [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'a8c27fc2-52cd-499c-b648-0355ed5ca72a', url: 'https://github.com/webex/react-ciscospark']]]
+        }
+    }
+
+    stage('Install') {
+      steps {
+        withCredentials([
+          string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
+        ]) {
+          sh '''#!/bin/bash -ex
```
Comment (adamweeks) [inline Jenkinsfile.tab]: `-ex` makes it very verbose. `-e` is what we switched the main jenkinsfile to.

File: Jenkinsfile.tab [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            set +x
+            rm .nvmrc
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
+            ~/.nvm/nvm.sh
+            echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' >> .npmrc
+            source ~/.bashrc || true
+            source ~/.profile || true'''
+          sh '''#!/bin/bash -ex
```
Comment (bzang) [inline Jenkinsfile.tab]: Should this and the previous statement be combined into a single `sh` block?

File: Jenkinsfile.tab [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    stage('Test') {
+      steps {
+        withCredentials([
+            string(credentialsId: 'ddfd04fb-e00a-4df0-9250-9a7cb37bce0e', variable: 'CISCOSPARK_CLIENT_SECRET'),
+            usernamePassword(credentialsId: 'SAUCE_LABS_VALIDATED_MERGE_CREDENTIALS', passwordVariable: 'SAUCE_ACCESS_KEY', usernameVariable: 'SAUCE_USERNAME'),
+            string(credentialsId: 'CISCOSPARK_APPID_SECRET', variable: 'CISCOSPARK_APPID_SECRET'),
+        ]) {
+            sh '''#!/bin/bash -ex
+           source ~/.nvm/nvm.sh
```
Comment (bzang) [inline Jenkinsfile.tab]: *nitpick* indenting seems off here.

File: Jenkinsfile.tab [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  options {
+    timeout(time: 2, unit: 'HOURS')
+    timestamps()
+    ansiColor('xterm')
+  }
+
+  stages {
+    stage('Checkout') {
+        steps{
+            checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'a8c27fc2-52cd-499c-b648-0355ed5ca72a', url: 'https://github.com/webex/react-ciscospark']]]
```
Comment (bzang) [inline Jenkinsfile.tab]: Can we split this into multiple lines so it's easier to read?

File: Jenkinsfile.tab (hunk 2 above)
Comment (moriahmaney) [reply Jenkinsfile.tab]: I can't remember why I had them separate for the portals jobs, i can try it.

File: Jenkinsfile.tab (hunk 3 above)
Comment (moriahmaney) [reply Jenkinsfile.tab]: good catch, VS code isn't great at figuring out jenkinsfiles..

File: Jenkinsfile.tab (hunk 4 above)
Comment (moriahmaney) [reply Jenkinsfile.tab]: Will do.

**Comment by moriahmaney** [comment]
Manually closing as something went sideways in CI

**Pull request closed without merging**